README.md

```diff
@@ -175,16 +175,16 @@ The documentation is also available as manpages by running `man 5 configuration.
 
 ## Uninstalling
 
-To run the latest version of the uninstaller, you can run the following command:
+To use the uninstaller, you can run the following command:
```

Comment:
I’m not sure how I feel about swapping the recommendation preference as the uninstaller is still in a half finished state so the idea behind recommending the uninstaller from latest `master` is for when we do improve the uninstaller that they can get those benefits even if they’re running an old nix-darwin version

Reply:
Fair enough. The downside is that it won’t use their configured `environment.systemPath`, but I guess ideally the uninstaller should be hermetic enough that that shouldn’t matter? I can drop this change if you’d prefer.